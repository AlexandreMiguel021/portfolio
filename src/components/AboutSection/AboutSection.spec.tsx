import { render } from '@testing-library/react'
import { AboutSection } from '.'
import '@testing-library/jest-dom'

describe('AboutSection', () => {
	it('should render a AboutSection correctly', () => {
		const { container } = render(<AboutSection description={''} />)
		expect(container).toMatchSnapshot()
	})
})
