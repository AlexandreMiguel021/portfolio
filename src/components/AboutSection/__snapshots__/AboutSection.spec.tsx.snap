// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutSection should render a AboutSection correctly 1`] = `
<div>
  <section
    class="section"
    id="about"
  >
    <div
      class="container"
    >
      <h2
        class="heading"
        data-aos="fade-up"
      >
        Sobre mim
      </h2>
      <div
        class="wrapper"
        data-aos="fade-up"
      >
        <div
          class="text"
        />
        <div
          class="avatar"
        >
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27270%27%20height=%27270%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt="Ilustração de um jovem de óculos de grau, pensando com o dedo indicador no queixo"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
            <noscript />
          </span>
        </div>
      </div>
    </div>
  </section>
</div>
`;
